src/components/Modal/MapOperatorToStakingProviderModal/index.tsx

```diff
@@ -109,40 +164,37 @@ const MapOperatorToStakingProviderModal: FC<
         <BodyLg mt={"10"}>
           Choose an application to map the Operator Address:
         </BodyLg>
-        <Box
-          p={"24px"}
-          border={"1px solid"}
-          borderColor={"gray.100"}
-          borderRadius={"12px"}
-          mt={"5"}
-          mb={"5"}
-        >
-          {isOperatorMappedOnlyInRandomBeacon ? (
-            <LabelSm>tBTC app</LabelSm>
-          ) : isOperatorMappedOnlyInTbtc ? (
-            <LabelSm>random beacon app</LabelSm>
-          ) : (
-            <LabelSm>tBTC + Random Beacon apps (requires 2txs)</LabelSm>
-          )}
-          <StakeAddressInfo stakingProvider={account ? account : AddressZero} />
-          <MapOperatorToStakingProviderForm
-            innerRef={formRef}
-            formId="map-operator-to-staking-provider-form"
-            initialAddress={
-              isOperatorMappedOnlyInRandomBeacon
-                ? mappedOperatorRandomBeacon
-                : isOperatorMappedOnlyInTbtc
-                ? mappedOperatorTbtc
-                : ""
-            }
-            onSubmitForm={onSubmit}
-            checkIfOperatorIsMappedToAnotherStakingProvider={
-              checkIfOperatorIsMappedToAnotherStakingProvider
-            }
-            mappedOperatorTbtc={mappedOperatorTbtc}
-            mappedOperatorRandomBeacon={mappedOperatorRandomBeacon}
-          />
-        </Box>
+        {isEmptyOrZeroAddress(mappedOperatorTbtc) && (
+          <Box
+            p={"24px"}
+            border={"1px solid"}
+            borderColor={"gray.100"}
+            borderRadius={"12px"}
+            mt={"5"}
+            mb={"5"}
+          >
+            <LabelSm>
+              {isEmptyOrZeroAddress(mappedOperatorTbtc)
+                ? "tBTC"
+                : isEmptyOrZeroAddress(mappedOperatorRandomBeacon)
+                ? "Random Beacon"
+                : "tBTC & Random Beacon"}
```

Comment:
Let's rever it to previous implementation, because it shows wrong values right now:

```suggestion
          {isOperatorMappedOnlyInRandomBeacon ? (
            <LabelSm>tBTC app</LabelSm>
          ) : isOperatorMappedOnlyInTbtc ? (
            <LabelSm>random beacon app</LabelSm>
          ) : (
            <LabelSm>tBTC + Random Beacon apps (requires 2txs)</LabelSm>
          )}
```

Reply:
https://github.com/threshold-network/token-dashboard/pull/625/commits/d1c18620ceb3a34e1ae7ab4a5ed176470eace2a6

Reply:
https://github.com/threshold-network/token-dashboard/pull/625/commits/ed0535912603406365af2a8e3f4457b5fce39d72